tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -949,7 +949,10 @@ def concat_func(*inputs):
         op = tf.concat(list(inputs), axis)
         return op
 
-    _compare_tvm_with_tflite(concat_func, shapes, accel_type)
+    # TODO(lhutton1) For now output is not bit exact with TFLite.
+    # This is because TFLite reference kernels are not being used.
+    # For this, TFLite will need upgrading to 2.6.
+    _compare_tvm_with_tflite(concat_func, shapes, accel_type, output_tolerance=1)
```

Comment:
Why is this changed now ?

Reply:
`qnn.concatenate` is lowered to a concatenate+requantize. Previously the requantize would not be offloaded, but now it is. There is a numerical inaccuracy in this case using the current version of TFLite, but we should see this issue when we upgrade to TFLite 2.6 and use the reference kernels that are made available.

Reply:
Thinking about it, I'm quite surprised that previously the concat on U55 + requantize on CPU yielded a bit exact result...